app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide :title, t("login.title") %>
+<title><%= t "login.title" %></title>
+<h1><%= t "login.title" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+      <%= f.label :email %>
```

Comment:
label tag alo nessacery use `I18n`
follow link https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models